prot.c

```
@@ -1,11 +1,29 @@
+#include "prot.h"
```

Better after the `<>` includes?

---

The idea is that this enforces prot.h to be self-contained. It will fail to build if it misses a type or something similar. If you put it after the system headers they could provide the missing types and hide the error.

---

Makes sense. Let's try to do this in more places.